airbyte-config/models/src/test/java/io/airbyte/config/helpers/KubeLoggingConfigTest.java

```diff
@@ -31,7 +31,8 @@
   public void cleanUpLogs() {
     if (logPath != null) {
       try {
-        LogClientSingleton.deleteLogs(new EnvConfigs(), logPath);
+        final EnvConfigs envConfigs = new EnvConfigs();
```

Comment:
what is the value of continuing to use this class in these tests? can we just remove it and create a couple static fields? given that this is fetching values from the environment it means the we have poor visibility into what the inputs to this test actually are if we use it.

Reply:
Yeah, there's work to be done if we want the logging config tests (there are several) to stop relying on env vars altogether.  Right now they're being triggered by a separate bash script, that does actually pass in env vars to them, but this is not at all obvious when you're reading the Java code.  And it's really opaque to figure out what they're being passed for actual values.  I hope to do a follow-up refactor around logging configuration, and could wrap this in with those.

Reply:
One thing to highlight for the refactor is we should not be doing mocking for Cloud Storage tests. All the mocking libraries do not properly mock the SDK (except for Minio with S3), and the recommended way for doing so is to use a real bucket. We can rejigger these to not be accepting env vars, but we should still preserve the intent to test against an actual bucket.